google/resource_compute_network_peering.go

```diff
@@ -61,6 +63,22 @@ func resourceComputeNetworkPeering() *schema.Resource {
 	}
 }
 
+func resourceComputeNetworkPeeringImporter(d *schema.ResourceData, meta interface{}) ([]*schema.ResourceData, error) {
+	config := meta.(*Config)
+	if err := parseImportId([]string{"(?P<network>[^/]+)/(?P<name>[^/]+)"}, d, config); err != nil {
+		return nil, err
+	}
+
+	// Replace import id for the resource id
+	id, err := replaceVars(d, config, "{{name}}")
```

Comment:
randomly pass by, (also looking for this feature), it appears to me the format of `id` for `google_compute_network_peering` is `network/name`.

I tried to checkout to this branch and was able to import my resource `google_compute_network_peering`, but the generated `id` seems only contains `name`.

Dunno if that matters, but wanna bring it up to discuss. Thanks in advance.